src/block.js

```diff
@@ -1,12 +1,39 @@
 /**
  * External dependencies
  */
+import { getColorClassName } from './util';
 import { __, sprintf } from '@wordpress/i18n';
+import classnames from 'classnames';
+
+export default function Block( {
+	color,
```

Comment:
What's color here?

Reply:
I must have missed this when cleaning up unused variables. I just removed it.